docs/application/web/api/6.5/device_api/mobile/tizen/se.html

```diff
@@ -1514,6 +1560,14 @@ <h2 id="api-features">2. Related Feature</h2>
            </div></p>
 <li class="feature">http://tizen.org/feature/network.secure_element</li>
 </div>
+<div class="def-api-feature">
+<p><div class="description">
+            <p>
+To guarantee this application will run on a device which supports the USB secure element feature, add the below feature declaration to the config file:
```

Comment:
To guarantee? 
Can we reword to something like
To ensure this application can run on a device...

Reply:
We use this exact wording "To guarantee this application will run..." in several API references (or even all of them) - we used it here to be consistent with other docs.

Do you think, we should change it?

Reply:
If this has been used consistently and you want to use the same let's keep it that way.